src/SpeedDial/SpeedDial.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode, useState } from "react";
+import Add from "../icons/Add";
+import { ButtonGroup } from "../ButtonGroup";
+import { st, classes } from "./speeddial.st.css";
+import { TooltipButton } from "../TooltipButton";
+
+interface SpeedDialProps {
+  children: ReactNode;
+  tooltipTitle: string;
+}
+
+function SpeedDial(props: SpeedDialProps) {
+  const [speedDialOpen, triggerSpeedDial] = useState("false");
+  const { children, tooltipTitle } = props;
+
+  const addButton = (
+    <TooltipButton
+      buttonProps={{
+        onBlur: () => triggerSpeedDial("false"),
+        onClick: () =>
```

Comment:
Also please use onPress, it's the preferred method overall because it's the event that aria hook provides...